### ISSUE TITLE
Is there a way to set maxAge or expiration for cookies?

### ISSUE THREAD

**Issue by jacobbubu** [senecajs/auth-token-cookie#10]

**Comment by mirceaalexandru** [comment]
Hey @jacobbubu,

Seneca-auth is using https://github.com/hapijs/hapi-auth-cookie for cookie authentication. There is a ttl option that can be set to control the expire time for cookies.

**Comment by wzrdtales** [comment]
@mirceaalexandru Yes this is right, but currently it does not seem to be possible to set all necessary options:
https://github.com/Concorda/auth-token-cookie/blob/master/lib/hapi-token-cookie.js#L41-L45

Is this intended?
